src/Nethermind/Nethermind.DataMarketplace.Initializers/NdmInitializer.cs

```diff
@@ -158,8 +158,20 @@ public NdmInitializer(INdmModule ndmModule, INdmConsumersModule ndmConsumersModu
             var webSocketsModule = webSocketsManager.GetModule("ndm");
             var notifier = new NdmNotifier(webSocketsModule);
             var ethRequestService = new EthRequestService(ndmConfig.FaucetHost, logManager);
+            if (baseDbPath.EndsWith("/") || baseDbPath.EndsWith("\\"))
```

Comment:
what if there are many? also I think this is a value somewhere in Path or Environment